dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/mysql/dolphinscheduler_ddl_post.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+alter table t_ds_process_definition drop primary key, ADD PRIMARY KEY (`id`,`code`);
```

Comment:
We should rename this file to `dolphinscheduler_ddl.sql`.

Reply:
When upgrading 2.0, the function of this file is to delete the original index and create a new index after splitting the json. The upgrade program has always included this logic.

Reply:
I don't quite understand what you mean. What I mean is rename this file.

Reply:
![image](https://user-images.githubusercontent.com/6930421/190333180-a1c69925-954f-4421-8258-41a14716791d.png)
A dolphinscheduler_ddl.sql file already exists,and they don't work the same way,this file needs to be executed after splitting the JSON.